tools/pd-ctl/pdctl/command/scheduler.go

```diff
@@ -513,6 +522,9 @@ func listSchedulerConfigCommandFunc(cmd *cobra.Command, args []string) {
 	p := cmd.Name()
 	if p == "list" {
 		p = cmd.Parent().Name()
+		log.Info("The list command will be deprecated in future version, use show command instead")
```

Comment:
Can we directly print instead of using log?

Reply:
like `cmd.Println`?

Reply:
As discussed with @rleungx, print message directly is similar to adding `Deprecated` field to the command which could break a script if it only expects the json output. 
```
The list command will be deprecated in future version, use show command instead
```
It is better to not change the command behavior before deprecate it, thus `log.Info` should be removed as well.